### ISSUE TITLE
[META] Accessibility Issues

### ISSUE THREAD

**Issue by wesruv** [SchoolofComputerScience/scs-app#54]
* Make sure all links have an obvious `:active` state
  * Some main menu items are very subtle
  * Cards are a little too subtle, IMO, maybe use a `outline` halo?
* Site needs to be keyboard navigatable
  * Navigating main menu with the keyboard is currently confusing, you can tab to hidden menu items and dropdown does not open
* Main menu toggle has no text for non-sighted users
* Form labels appear to be missing on text inputs (they don't have to be visible to sighted users, just need to exist)
* Some search forms have buttons with no text (e.g. Directory)
* Fair number of pages are missing headings or h1's

**Comment by wesruv** [comment]
I was going to work on making the menu keyboard navigatable, shouldn't be too hard, just have to expand some of the mouse logic to work with focus.

**Comment by kylemorelock** [comment]
@wesruv, are these issues still issues?

**Comment by wesruv** [comment]
Menu issues are fixed

Form labels are still an issue (See WAVE Accessibility plugin for whatever browser):
![image](https://user-images.githubusercontent.com/5607236/33388358-c30d609a-d4fd-11e7-89bf-3a104cbc308d.png)

Same with empty button text

Directory is missing an H1 as well as a lot of other pages. Not having a headline is an accessibility issue, so the design needs to support that somehow. I believe there's a design that tries to address how headlines could look on a card driven page... I'll try to track it down.

**Comment by wesruv** [comment]
Here's the mockup of what that could look like, it was a quick and dirty thing:
![image](https://user-images.githubusercontent.com/5607236/33389296-65216140-d500-11e7-804b-a35854ca092b.png)
